stmt.go

```diff
@@ -79,6 +79,10 @@ func prepareSpannerStmt(q string, args []driver.NamedValue) (spanner.Statement,
 			name = names[i]
 		}
 		ss.Params[name] = v.Value
+		// check if value is uint change it to int64
+		if _, ok := v.Value.(uint); ok {
```

Comment:
Why do we need this special handling here? Is it because all the other types that we support in `database/sql` are also supported in the Go client, except for this type? If so; Would it not make more sense to add support for `uint` to the Go client, so the support fits more naturally here?

Reply:
Removed